src/app/shipping/BillingSameAsShippingField.tsx

```diff
@@ -17,7 +17,7 @@ const BillingSameAsShippingField: FunctionComponent<BillingSameAsShippingFieldPr
     return <CheckboxFormField
         id="sameAsBilling"
         labelContent={ labelContent }
-        name="billingSameAsShipping"
+        name="isBillingSameAsShipping"
```

Comment:
Is this change intentional? I believe it's going break any customisation that targets `[name=billingSameAsShipping]` so it's better not to change it. You can keep the name of the variable as `isBillingSameAsShipping` in other places, but not this one as it can affect the HTML output.

Reply:
Good point. I've changed this back